service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateLogAnalyticsWorkspaceStep.java

```diff
@@ -58,21 +54,25 @@ public StepResult undoStep(FlightContext context) throws InterruptedException {
               .toList();
       solutions.forEach(
           s -> {
-            armManagers.azureResourceManager().genericResources().deleteById(s.id());
-            logger.info(
-                "{} resource with id={} deleted.",
-                AzureResourceTypeUtils.AZURE_SOLUTIONS_TYPE,
-                s.id());
+            try {
+              armManagers.azureResourceManager().genericResources().deleteById(s.id());
+              logger.info(
+                  "{} resource with id={} deleted.",
+                  AzureResourceTypeUtils.AZURE_SOLUTIONS_TYPE,
+                  s.id());
+            } catch (ManagementException e) {
```

Comment:
Why did you decide to nest a try/catch here instead of allowing the handling down at line 70 to do the logging?